packages/app/playwright.config.ts

```diff
@@ -33,7 +38,7 @@ export default defineConfig({
      * Maximum time expect() should wait for the condition to be met.
      * For example in `await expect(locator).toHaveText();`
      */
-    timeout: 5000,
+    timeout: 8000,
```

Comment:
Were there any problems with the 5000 ms?

Reply:
Good point, it was for experimental reasons. Changed!